### DISCUSSION TITLE
DVR dietro nat

### DISCUSSION THREAD

**Discussion by danye72** [indomus/forum#1840] [Networking]
Un saluto a tutti ho seguito la splendida guida su indomus per accedere ad homeassistant su una connessione nattata. Solo che al posto di HA devo accedere ad un DVR con indirizzo ip 192.168.8.201:8081. La vpn funziona e assegna l'indirizzo 10.8.0.2 al client. Sempre sul server remoto eseguo caddy che gira le richieste in arrivo sul client vpn 10.8.0.2:8081. Poi però dall'esterno non riesco a raggiungere il DVR. Qualche suggerimento?
Premetto che dal server virtuale riesco a pingare il client vpn correttamente quindi la von funziona

**Comment by snakuzzo** [comment]
Devi creare la regola su iptables per il pre-routing...solo che al posto della porta che hai usato per collegarti ad HA tramite Caddy sul RPI, userai la porta per collegarti al DVR tramite Caddy sul RPI

**Reply by danye72** [reply]
Grazie Diego, quindi c'è bisogno di una regola sul server virtuale? Riesci a farmi un esempio del comando da inviare?
Premetto che non uso un Raspberry come client VPN ma un gl.inet
